hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/MerkleHederaState.java

```diff
@@ -238,6 +242,30 @@ public int getVersion() {
         return CURRENT_VERSION;
     }
 
+    /**
+     * To be called ONLY at node shutdown. Attempts to gracefully close any virtual maps. This method is a bit of a
+     * hack, ideally there would be something more generic at the platform level that virtual maps could hook into
```

Comment:
Might be good to add an issue now to get that created, so it doesn't get lost.

Reply:
https://github.com/hashgraph/hedera-services/issues/8278